integration-tests/actions/actions.go

```diff
@@ -336,3 +339,15 @@ func EncodeOnChainExternalJobID(jobID uuid.UUID) [32]byte {
 	copy(ji[:], strings.Replace(jobID.String(), "-", "", 4))
 	return ji
 }
+
+// GetTestLogger instantiates a logger that takes into account the test context and the log level
+func GetTestLogger(t *testing.T) zerolog.Logger {
```

Comment:
I actually think this might make more sense to add to the chainlink-env environment or config modules.

Reply:
@skudasov what do you think?

Reply:
Imo we can just have `logger.go` in each repo.